modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/display/context/BaseSectionDisplayContext.java

```diff
@@ -89,9 +113,12 @@ protected String getAddStructuredContentItemURL(long objectDefinitionId) {
 		return sb.toString();
 	}
 
+	protected abstract String getCMSSectionFilter();
```

Comment:
Question, Any reason  I  do not get to make this abstract and to implement here the common (`return StringPool.BLANK;`) as other methods on this same class?